presto-native-execution/README.md

```diff
@@ -96,17 +86,100 @@ checks.  When we need one it can be added.
 
 ## Development Env.
 
-### Setting up on macOs
-
-See `scripts/setup-macos.sh`
-
-After running the setup script add the cmake-format bin to your $PATH, maybe
-something like this in your ~/.profile:
+### Basic setup on MacOs
 
+* Clone the whole Presto repository.
+* Run `presto/presto-native-execution/scripts/setup-macos.sh`.
+* Add the cmake-format bin to your $PATH, maybe something like this
+  in your ~/.profile:
 ```
 export PATH=$HOME/bin:$HOME/Library/Python/3.7/bin:$PATH
 ```
-
-### Setting up on Linux (CentOs 8 or later)
-
-See `scripts/setup-linux.sh`
+* It is recommended to use some sort of IDE on MacOs.
+  [CLion](https://www.jetbrains.com/clion/) is an option.
+
+### Basic setup on Linux (CentOs 8 or later)
+
+* Clone the whole Presto repository.
+* Run `presto/presto-native-execution/scripts/setup-linux.sh`
+
+### Setting up local Presto environment on MacOs (with optional IntelliJ and CLion) 
```

Comment:
nit: Looks like macOS and MacOS are conventional spellings: https://en.wikipedia.org/wiki/MacOS

Reply:
Potato, Potato... :)
I can change this.